worker/globalclockupdater/worker.go

```diff
@@ -24,7 +24,7 @@ type Logger interface {
 // Config contains the configuration for the global clock updater worker.
 type Config struct {
 	// NewUpdater returns a new global clock updater.
-	NewUpdater func() (globalclock.Updater, error)
```

Comment:
Including this error was a hangover from Mongo leases. It hasn't been necessary since we switched to Raft-only.